other/materials_designer/create_interface_with_relaxation_ase_emt.ipynb

```diff
@@ -4,136 +4,145 @@
    "cell_type": "markdown",
```

Comment:
<p>This needs to be updated</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>